test-cypress/integration/sparql/sparql.menu.spec.js

```diff
@@ -703,7 +717,7 @@ describe('SPARQL screen validation', () => {
         });
 
         it('Test URL to current query', () => {
-            const query = 'SELECT ?s ?p ?o WHERE {?s ?p ?o .} LIMIT 100';
+            const query = 'SELECT ?sub ?pred ?obj WHERE {?sub ?pred ?obj .} LIMIT 100';
```

Comment:
but why this was changed? :)

Reply:
Because this is the default query but on single line. I wanted it to be more unique just to be sure.